src/elements/DropdownSelect.tsx

```diff
@@ -19,6 +19,7 @@ export interface DropdownSelectProps
   readonly errorMessage?: string;
   readonly handleChange?: (newValue: string) => void;
   readonly label: string;
+  readonly isOptional?: boolean;
```

Comment:
Is this missing the add to the actual component or will that occur after the merge?

Reply:
This will occur after the merge. I realized that the actual code that does the actual optional functionality is part of the other PR. 🥲